tests/test.py

```diff
@@ -47,6 +47,7 @@ def test_metadata():
     assert metadata['name'] == 'lm_1b TensorFlow Model'
     assert metadata['description'] == 'Generative language model trained on the One Billion Words data set'
     assert metadata['license'] == 'Apache v2'
+    assert metadata['source'] == 'https://developer.ibm.com/exchanges/models/all/max-news-text-generator/'
 
 
 @pytest.mark.skipif("TRAVIS" in os.environ, reason="test runs out of memory on Travis-CI")
```

Comment:
Since we know the test will fail, should we add a test that verifies the OOM error handling?

Reply:
The tests is still useful when testing locally and on Jenkins. Travis tests everything in a container and when that container runs out of memory I believe everything gets killed

Reply:
Right, but I meant can we add a test that only runs on Travis that verifies that the OOM condition is properly handled? I'm not sure what would happen in that scenario, since you are catching the error. Is the container going down (resulting in HTTP code 5xx) or will the prediction include an error status?

Reply:
The container that is going down is the Travis container containing both the test scripts and the MAX container. Once the Travis container goes down we have no way to test anything. I will push a test commit to verify this is the case

Reply:
I was wrong. The Travis Container is not failing

```
_________________________________ test_predict _________________________________

    @pytest.mark.skipif("TRAVISxxxxxxxxx" in os.environ, reason="test runs out of memory on Travis-CI")

    def test_predict():

        model_endpoint = 'http://localhost:5000/model/predict'

        file_path = 'samples/sample1.txt'

    

        with open(file_path, 'rb') as file:

            file_form = {'text': (file_path, file, 'text/plain')}

            r = requests.post(url=model_endpoint, files=file_form)

    

>       assert r.status_code == 200

E       assert 500 == 200

E        +  where 500 = <Response [500]>.status_code

tests/test.py:62: AssertionError

------------------------------ Captured log call -------------------------------

connectionpool.py          206 DEBUG    Starting new HTTP connection (1): localhost:5000

connectionpool.py          396 DEBUG    http://localhost:5000 "POST /model/predict HTTP/1.1" 500 37

===================== 1 failed, 2 passed in 32.77 seconds ======================

The command "pytest tests/test.py" exited with 1.
```